packages/embeds/embed-core/src/embed.ts

```diff
@@ -766,13 +795,16 @@ class CalApi {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Queue = any[];
```

Comment:
A queue basically holds the arguments to all the instruction functions, so it could be anything.